SCVersion.txt

```diff
@@ -2,7 +2,7 @@
 # Note that you need to "make install" for this information to be copied into all places.
 
 set(SC_VERSION_MAJOR 3)
-set(SC_VERSION_MINOR 13)
+set(SC_VERSION_MINOR 14)
```

Comment:
Note sure what this is about

Reply:
Yeah, sorry about that. When you get a chance, please remove these two commits.